File: examples/ios-host-shell/HostShell/Sources/Modules/Contacts/ContactsModule.swift
import Foundation

struct ContactsModule: DDBModule {
    static let tables = ["contact"]

    static func bootstrap(_ driver: DoogatDriver) throws {
        _ = try driver.executeSql(sql: """
            CREATE TABLE IF NOT EXISTS contact (
                name TEXT NOT NULL,
                relationship TEXT,
                email TEXT
            )
        """)
    }
}

File: examples/android-host-shell/feature-contacts/src/main/java/com/doogat/hostshell/contacts/ContactsModule.kt
package com.doogat.hostshell.contacts

import com.doogat.hostshell.DDBModule
import uniffi.ddb_core.DoogatDriver

object ContactsModule : DDBModule {
    override val tables = listOf("contact")

    override fun bootstrap(driver: DoogatDriver) {
        driver.executeSql("""
            CREATE TABLE IF NOT EXISTS contact (
                name TEXT NOT NULL,
                relationship TEXT,
                email TEXT
            )
        """.trimIndent())
    }
}
